index.html

```diff
@@ -10,6 +10,18 @@
 </head>
 
 <body>
+    <div class="privacy">
+<h1>We don't store your<br>
```

Comment:
We can change this h1 with h2. Because we should use only one h1 tag per page for best practices and it should be page title.

Reply:
for sure. I can change it now if you want.